astropy/time/tests/test_methods.py

```diff
@@ -619,6 +629,91 @@ def test_sort(self, use_mask):
             assert np.all(self.t0.sort(-1)[:, :, 0] == self.t0.min(-1))
             assert np.all(self.t0.sort(-1)[:, :, -1] == self.t0.max(-1))
 
+    @pytest.mark.parametrize('axis', [None, 0, 1, 2, (0, 1)])
+    @pytest.mark.parametrize('where', [True, np.array([True, False, True, True, False])[..., np.newaxis]])
+    def test_mean(self, use_mask, axis, where):
+        self.create_data(use_mask)
+
+        kwargs = dict(axis=axis, where=where, keepdims=True)
```

Comment:
Maybe the easiest would be to parametrize on `keepdims` for `False`, `True`? Could even skip some parts of the tests for `False` if that helps.